charts/cluster-scanner/templates/deployment.yaml

```diff
@@ -0,0 +1,398 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "cluster-scanner.fullname" . }}
+  labels:
+   {{- include "cluster-scanner.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "cluster-scanner.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "cluster-scanner.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "cluster-scanner.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: rsi
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+      {{- with .Values.rsi }}
+          image: "{{ .image.repository }}:{{ .image.tag }}"
+          imagePullPolicy: {{ .image.pullPolicy }}
+          ports:
+            - name: metrics
+              containerPort: {{ .ports.metrics }}
+              protocol: TCP
+            - name: probes
+              containerPort: {{ .ports.probes }}
+              protocol: TCP
+          env:
```

Comment:
Is it possible to get rid from ENVs and read informations from  a file? This is specially useful in case of information that should not be disclosed (access keys, password, ..) like in `SYSDIG_ACCESS_KEY`, `ANALYZER_CACHE_REDIS_PASSWORD`, ...

Reply:
atm no, there is no such capability in the services. We could try to add this effort later on 

cc @michele-mangili @guidobonomi

Reply:
We have a task in the product backlog to address this in upcoming releases https://sysdig.atlassian.net/browse/SSPROD-24679 . It may also be beneficial to reduce the number of env vars we need to set, and simplify the overall configuration of the cluster-scanner.